tests/testthat/test-avg_profile_ranking_best_case_worst_case.R

```diff
@@ -0,0 +1,67 @@
+test_that("`NA` in `emission_profile` and `profile_ranking` gives correct values for `avg_profile_ranking_best_case` and `avg_profile_ranking_worst_case`", {
+  example_data <- example_best_case_worst_case_profile_ranking_product_level(
+    emission_profile = c("low", "low", NA_character_),
+    profile_ranking = c(1.0, 2.0, NA)
+  )
+
+  out <- create_avg_profile_ranking_best_case_worst_case_at_product_level(example_data)
+
+  expected_best_case <- c(3 / 2, NA)
+  expected_worst_case <- c(3 / 2, NA)
```

Comment:
So the best case and the worst case are the same? Maybe we need a different example to test they *can* be different.

Reply:
You can check in the above reprex that the best case and worst case can be different. For example: `avg_profile_ranking_best_case` and `avg_profile_ranking_worst_case` are different in the above reprex. There is no requirement to make them different for this test, hence I will not change the example. :)